geniza/footnotes/admin.py

```diff
@@ -140,6 +144,8 @@ def get_queryset(self, request):
                     "authorship__creator__last_name", "authorship__creator__first_name"
                 ),
             )
+            .select_related("source_type")
+            .prefetch_related("authorship_set", "authorship_set__creator")
```

Comment:
i think that since each `Authorship` has only one `Creator`, you can use the same technique as here to push one query onto the db side:
https://github.com/Princeton-CDH/geniza/blob/58586d5ff8e8325d7bb852d7785737caa23fee32/geniza/corpus/admin.py#L235-L250

so it'd be:
```py
.prefetch_related(Prefetch("authorship_set", queryset=Authorship.objects.select_related("creator")))
```

idk if it'll make any difference, just an idea